Resources/ArmTemplates/template.json

```diff
@@ -473,22 +473,6 @@
                 "name": "AppUISettings__WhereNextComponentSettings__TextFieldMaxChars",
                 "value": "500"
               },
-              {
-                "name": "OnspdSearchSettings__SearchServiceName",
-                "value": "[parameters('searchDomain')]"
-              },
-              {
-                "name": "OnspdSearchSettings__SearchServiceAdminApiKey",
-                "value": "[parameters('searchAdminKey')]"
-              },
-              {
-                "name": "OnspdSearchSettings__SearchServiceQueryApiKey",
-                "value": "[parameters('searchQueryKey')]"
-              },
-              {
-                "name": "OnspdSearchSettings__IndexName",
```

Comment:
Is it worth keeping this configurable if we need to change indexes over in future?

Reply:
There's a separate (existing) config entry for the index name